app/helpers/comments_helper.rb

```diff
@@ -291,4 +291,37 @@ def fallback_url_for_comment(comment, options = {})
 
   end
 
+  # find the parent of the commentable
```

Comment:
@sarken Oh, is this because you don't actually have the comment itself? If so, ignore. :)